CHANGES.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Rename `GeometryInterface` struct to `Geometry` - that is the last rename of this struct - promised.
```

Comment:
:shipit:

Reply:
Yes, its is impossible to pack everything into a name ;)